limitations.md

```diff
@@ -23,8 +23,8 @@ Setting up a Horizon server under Windows has not been extensively tested.
 
 ## Limitations
 
+* Collection names may not start with  `hz_`. Using this prefix will result in an error.
```

Comment:
I think this should move off of the limitations page and go to the collections page as a note. This isn't a limitation like the others where we expect to fix it, it's just how horizon works now